shotover-proxy/src/frame/cassandra.rs

```diff
@@ -93,7 +95,9 @@ impl CassandraFrame {
     // Count the amount of queries in this `CassandraFrame`, this will either be the count of all queries in a BATCH statement or 1 for all other queries
     pub(crate) fn get_message_count(&self) -> Result<NonZeroU32> {
         Ok(match &self.operation {
-            CassandraOperation::Batch(bytes) => get_batch_len(bytes)?,
+            CassandraOperation::Batch(batch) => {
+                NonZeroU32::new(batch.queries.len() as u32).unwrap_or(nonzero!(1u32))
```

Comment:
Similar to the bug-fix PR, can we add a comment explaining this? i.e. BATCH statements query length can be 0 and for our purposes counting as 1 is fine.